filebeat/module/elasticsearch/deprecation/test/test-json.log-expected.json

```diff
@@ -1,5 +1,6 @@
 [
     {
+        "@timestamp": "2019-01-30T14:16:20.233Z",
```

Comment:
Good news: the `@timestamp` field is back.

Bad news: if you compare the value of this field vs. the value in the [raw log line](https://github.com/elastic/beats/blob/1e3b7ef50ffbf1edd63e989d5d110fbf00dc52b8/filebeat/module/elasticsearch/deprecation/test/test-json.log#L1) that generated it, you'll notice that the time zone in this field, `Z`, is incorrect. It should be `-0800` instead. Alternatively, the time here should be `22:16:20.233Z`.

Reply:
Same goes for all other `@timestamp` fields in this PR that are being generated from timestamps  **with time zones** in the raw logs.

Reply:
We definitively need support back for the time zones.